src/Uno.UWP/ApplicationModel/Chat/ChatMessageManager.iOS.cs

```diff
@@ -0,0 +1,43 @@
+﻿#if __IOS__
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MessageUI;
+using UIKit;
+using Windows.Foundation;
+
+namespace Windows.ApplicationModel.Chat
+{
+	public partial class ChatMessageManager
+	{
+		public static IAsyncAction ShowComposeSmsMessageAsync(ChatMessage message)
+		{
+			if (message == null)
+			{
+				throw new ArgumentNullException(nameof(message));
+			}
+
+			var window = UIApplication.SharedApplication.KeyWindow;
+			var controller = window.RootViewController;
+
+			var messageController = new MFMessageComposeViewController();
+
+			messageController.Body = message.Body;
+			messageController.Recipients = message?.Recipients?.ToArray() ?? new string[] { };
```

Comment:
Same question here. From the [docs](https://developer.apple.com/documentation/messageui/mfmessagecomposeviewcontroller/1614071-recipients) the Apple API seems to be more restricted:

> Each string in the array should contain the phone number of the intended recipient.

Reply:
Similarly on iOS it outputs non-numeric recipients as normal string in the _To_ field